docs/project/contributing.md

```diff
@@ -53,7 +53,7 @@ $ brew install bun
 
 ## Install LLVM
 
-Bun requires LLVM 16 and Clang 16 (`clang` is part of LLVM). This version requirement is to match WebKit (precompiled), as mismatching versions will cause memory allocation failures at runtime. In most cases, you can install LLVM through your system package manager:
+Bun requires LLVM 16/Clang 16 or LLVM 17/Clang 17 (`clang` is part of LLVM). This version requirement is to match WebKit (precompiled), as mismatching versions will cause memory allocation failures at runtime. In most cases, you can install LLVM through your system package manager:
```

Comment:
remove 16. it will no longer work, explained by what follows:

> This version requirement is to match WebKit (precompiled), as mismatching versions will cause memory allocation failures at runtime.